syne_tune/optimizer/schedulers/neuralbands/networks.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+
+import numpy as np
+import argparse
+import pickle
+import os
+import time
+import torch
+import pandas as pd 
+import scipy as sp
+import torch.nn as nn
+import torch.optim as optim
+
+
+if torch.cuda.is_available():  
+    dev = "cuda:0" 
+else:  
+    dev = "cpu" 
+device = torch.device(dev)
+
+class Network_exploitation(nn.Module):
+    def __init__(self, dim, hidden_size=100):
+        super(Network_exploitation, self).__init__()
+        self.fc1 = nn.Linear(dim, hidden_size)
+        self.activate = nn.ReLU()
+        self.fc2 = nn.Linear(hidden_size, hidden_size)
+        self.fc3 = nn.Linear(hidden_size, 1)
+    def forward(self, x1, b):
+        f1 = self.activate(self.fc1(x1))
+        f2 = self.activate(self.fc2(b * f1))
+        return self.fc3(f2)
+    
+    
+class Exploitation:
+    def __init__(self, dim, lr = 0.001, hidden=100):
```

Comment:
Could you annotate types? https://towardsdatascience.com/type-annotations-in-python-d90990b172dc